src/pages/utilities/config.md

```diff
@@ -0,0 +1,68 @@
+---
+previousText: ''
+previousUrl: ''
+nextText: ''
+nextUrl: ''
+---
+
+# Config
+
+The Config lets you configure your entire app. You can set the tab button layout, icon mode, animations, and more here.
+
+## Usage
+
```

Comment:
Maybe here we should do something similar to components where there is a Basic Usage like:

```
### Basic example

(add some usage here about changing string or boolean types)

### Customizing Animations

(add some usage here on changing an overlay animation)
```